handbook/ce/customer_issues.md

```diff
@@ -0,0 +1,32 @@
+# Filing customer issues
+
+Read [the support overview](index.md) before filing an issue.
+
+## Create a customer issue
+
+Customer support tickets should be translated to GitHub issues. In such cases, [create a
+new issue for the request](https://github.com/sourcegraph/customer/issues/new).
```

Comment:
I don't think it makes sense to do this for *all* customer issues. For example, a vast majority of the issues that come in are solved by the Customer Engineering team at the point of contact (Slack, Jira, etc.) and creating a GitHub issue for all of these would be a lot of work and harm to _their_ workflow.

I would rephrase this to indicate that when a customer issue requires pulling in another engineering team (distribution or otherwise), then create a GitHub issue.

Reply:
Ill rephrase it to indicate when it requires other engineers. While its not the intention for all the issues they manage to get a GitHub issue associated, I think it would be a useful metric for CE to be able to analyze what should we document more or what are the frequent type of questions we get.

Reply:
@slimsag updated, please review

Reply:
Definitely agree with @slimsag . Thanks for updating the language!

If metrics were a priority, we definitely would not be using GitHub. We'd force all tickets to be entered into JIRA (or some replacement tool like ZenDesk) and use that for issue categorization and analysis. GitHub issues may be the last place I'd ever want to analyze issue/question frequency (or calculate any metrics, honestly) 😂 . 

Beyond the changes you made, though, I strongly disagree with one more part of this approach—I think we should be _aggressively_ minimizing the use of the sourcegraph/customer repo, and instead by default posting everything to sourcegraph/sourcegraph. Use of sourcegraph/customer means everyone on the team has to be aware of (and use) multiple trackers, there's no way for new users who encounter an issue to +1 it, devs may forget to close both issues (if mirrored), leading to issue debt, and more.

We open sourced our issue tracker for a reason—I'd prefer to have an internal goal of shutting down sourcegraph/customer, or of saying that sourcegraph/customer is NEVER used for discussion of an issue, and should instead only be a repository for customer data that we link out to from the actual issue filed in sourcegraph/sourcegraph. 

How would you respond to those concerns?

Reply:
I believe incidents are for the most part customer specific, a customer having a problem related in many cases to its particular environment or deployment. There might be a set of general issues to be created that are related and must be linked to the incident, and those should be done in `/sourcegraph` where they can be :+1: and publicly discussed.
This works in a similar way to incident response workflows, were the immediate issue is worked from the generated alert (and in some cases automated issue) and later on we create any follow-up issues for the permanent or general fixes.

The main concerns is that right now there does not seem to be a clear funnel and a lot of this flows through a Slack thread or conversation, or linked from Jira in Slack, which makes it impossible to get an overview of the current issues or past issues we worked on over an iteration. Additionally, issues in `/sourcegraph` because we cant label or add any customer information, its quite hard to group by customer and get an overview of a customer incidents.

I think the tradeoff of having 2 issues trackers in GitHub (one for most of our issues and one for customer specific ones) for the issues that engineering has to work on is fairly minimal, and creates a simple funnel for all incidents, which at present are distributed between many places (Slack, Jira, etc).
Ideally, we would have a single issue tracker, but unfortunately that is not possible when mixing customer and public incidents, and I think additionally having 2 trackers, makes it harder to leak information by incorrectly setting a message or setting an issue to private (if we had a tracker that allowed that).